### PR TITLE
assignment 9 directory added

### PR DESCRIPTION
**Pull request by prassr**
assignment 9 created.

@hashkrish

### CODE REVIEW

File: assignment_9/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    - ` petal_width `
+
+* ` jq ` parsable output :
+
+    ```json
+    {"mean":<mean>, "std":<std>}
+    ```
+
+* ` <mean> ` : mean of rows of ` <feature_name> ` over ` <class> `
+* ` <std> ` : standard deviation of rows of ` <feature_name> ` over ` <class> `
```
Comment (hashkrish) [inline assignment_9/README.md:79]: I think `mean` itself is enough. But you can ask them to include the class and feature_name in output json

File: assignment_9/info.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+5.006     3.428      1.462     0.246
+0.352     0.379      0.174     0.105
+5.936     2.770      4.260     1.326
+0.516     0.314      0.470     0.198
+6.588     2.974      5.552     2.026
+0.636     0.322      0.552     0.275
```
Comment (hashkrish) [inline assignment_9/info.txt:6]: Is this tab separated or space separated?

File: assignment_9/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+### Attribute Information
+
+1. sepal length in cm
+2. sepal width in cm
+3. petal length in cm
+4. petal width in cm
+5. class:
+    - Iris Setosa
+    - Iris Versicolour
+    - Iris Virginica
```
Comment (hashkrish) [inline assignment_9/README.md:20]: Include the hyphen between Iris and class name

**Review by hashkrish — CHANGES_REQUESTED** [review]
@prassr I have given my comment. Kindly check

File: assignment_9/evaluation_file.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+if ! [ -f ~/se2001/assignment_6/script.sh ]; then
+    echo "File $HOME/se2001/assignment_6/script.sh not found" >&2
+    exit 1
+fi
+
+if ! [ -x ~/se2001/assignment_6/script.sh ]; then
+    echo "File $HOME/se2001/assignment_6/script.sh is not executable" >&2
+    exit 1
+fi
```
[the remark this answers is not in the document]
Comment (prassr) [reply assignment_9/evaluation_file.sh:11]: I made a big mistake. Sorry, sir.

File: assignment_9/info.txt (hunk 2 above)
Comment (prassr) [reply assignment_9/info.txt:6]: Space separated values.

File: assignment_9/README.md (hunk 1 above)
Comment (prassr) [reply assignment_9/README.md:79]: So, they can solve it using sql aggregate functions. This is nice Idea. Thanks.

**Review by hashkrish — APPROVED** [review]

**Pull request merged**